nc_time_axis/__init__.py

```diff
@@ -36,6 +36,14 @@ def __init__(self, datetime, calendar):
         self.datetime = datetime
         self.calendar = calendar
 
+    def __eq__(self, other):
+        return (isinstance(other, self.__class__) and
+                self.datetime == other.datetime and
+                self.calendar == other.calendar)
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
```

Comment:
I'm not sure if this is good enough to define `__ne__` in terms of `__eq__`.
If not I could do something along the lines of 
```
    def __ne__(self, other):
        return (not isinstance(other, self.__class__) or
                self.datetime != other.datetime or
                self.calendar != other.calendar)
```